docs/deployment/containers/docker.rst

```diff
@@ -352,46 +368,46 @@ Persistent data directory
 -------------------------
 
 Docker containers are ephemeral, meaning that containers are expected to come
-and go, and any data inside them is lost when the container goes away. For this
-reason, it is required that you mount a persistent ``data`` directory on your
-host machine to the ``/data`` directory inside the container, like so::
+and go, and any data inside them is lost when the container is removed. For
+this reason, you should mount a persistent ``data`` directory on your host
+machine to the ``/data`` directory inside the container::
 
     $ docker run -d -v /srv/crate/data:/data crate \
         crate -Cnetwork.host=_site_
 
 Here, ``/srv/crate/data`` is an example path, and should be replaced with the
-path to your host machine ``data`` directory.
+path to your host machine's ``data`` directory.
 
 See the `Docker volume`_ documentation for more help.
 
 Custom configuration
 --------------------
 
-If you want to use a custom configuration, it is strongly recommended that you
-mount configuration files on the host machine to the appropriate path inside
-the container. That way, your configuration isn't lost when the container goes
-away.
+If you want to use a custom configuration, it is recommended that you mount
+configuration files on the host machine to the appropriate path inside the
+container. That way, your configuration will not be lost if the container is
+removed.
 
-Here's how you would mount ``crate.yml``::
+Here is an example of how you could mount the ``crate.yml`` config file::
 
     $ docker run -d \
         -v /srv/crate/config/crate.yml:/crate/config/crate.yml crate \
         crate -Cnetwork.host=_site_
 
 Here, ``/srv/crate/config/crate.yml`` is an example path, and should be
-replaced with the path to your host machine ``crate.yml`` file.
+replaced with the path to your host machine's ``crate.yml`` file.
 
 Troubleshooting
 ===============
 
 The official `CrateDB Docker image`_ ships with a liveness `healthcheck`_
 configured.
 
-This healthcheck will flag a problem when the CrateDB process might have
-crashed or hanged inside the container without terminating.
+This healthcheck will flag a problem if the CrateDB process crashed or hung
```

Comment:
"Health check" imho. Don't think it's conventional enough to have a compound without space.